Makefile

```diff
@@ -82,6 +82,9 @@ test: bazel-test
 fuzz:
 	hack/dockerized "./hack/fuzz.sh"
 
+integ-test:
```

Comment:
It seems that this script basically runs unit tests in privileged environment.

I think this can be achieved by using hack/dockerized with slight change that enable running it as  privileged container:
Add the following snippet just before [line 149](https://github.com/kubevirt/kubevirt/blob/main/hack/dockerized#L149), where the builder container starts.
In hack/dockerized:
```
if [ "${CONTAINER_PRIVILEGED}" == "true" ]; then
    privileged="--privileged --rm"
fi

$KUBEVIRT_CRI run ${privileged} ...
...
```

In the  Makefile: 
```
integ-test:
CONTAINER_PRIVILEGED=true hack/dockerized "go test -v ./pkg/network/driver/nmstate/... --run-integration-tests"
```

Reply:
I do not think it is a fit.
First, the dockerized one seems to use the host network (`--network host`) which is surely not what we want here.
Then we have this bazel server it depends on. I would prefer not to depend on it if possible.

If we want to optimize this further, I think it can be done as a follow up, for now it assists in testing the nmstate package pretty well. If the integration is found useful, we may want to customize dockerize to allow it to run such tests.